src/core/datatypes/forte_struct.cpp

```diff
@@ -59,6 +59,12 @@ void CIEC_STRUCT::setValue(const CIEC_ANY &paValue) {
   }
 }
 
+void CIEC_STRUCT::reset() {
+  for (size_t i = 0; i < getStructSize(); ++i) {
```

Comment:
I would suggest assigning `getStructSize()` to a local variable to keep it from getting called on every iteration:
```suggestion
  for (size_t i = 0, size = getStructSize(); i < size; ++i) {
```

Reply:
done